### DISCUSSION TITLE
Tutor account error after they receive "Instructor" Role

### DISCUSSION THREAD

**Discussion by mertkabadayii** [ls1intum/Artemis#7534] [Q&A] [answered]
After we gave our Tutors "Instructor" and "Editor" Roles, so they can change the Tutors on their Exercise Teams, they receive such error: They no longer can edit the teams or create a new one.
Tutors could change the Tutors attended for the Groups, after they receive "Instructor" Role. If there is a way to solve this, i would be happy if someone could help. Artemis Version: 6.6.3
![WhatsApp Image 2023-11-06 at 21 49 27_c6532f92](https://github.com/ls1intum/Artemis/assets/78808727/e064a6c0-1204-4a5d-8bb5-227d7e99e642)

**Comment by krusche** [comment, marked as answer]
Can you ask the tutors to logout and login again. This might fix the issues

**Reply by mertkabadayii** [reply]
Thanks for the Tip, it works now.
